core/store/migrations/migration1596021087/migrate.go

```diff
@@ -0,0 +1,26 @@
+package migration1596021087
+
+import (
+	"github.com/jinzhu/gorm"
+)
+
+// Migrate creates the encrypted_p2p_keys table and renames encryped_secret_keys to the more specific encrypted_vrf_keys
+func Migrate(tx *gorm.DB) error {
+	return tx.Exec(`
+		ALTER TABLE encrypted_secret_keys RENAME TO encrypted_vrf_keys;
```

Comment:
This could be split into two migrations. 
The reason is you're doing two separate things here: renaming encrypted_secret_keys to be more specific and adding a new encrypted_p2p_keys table. 
I would go so far as to suggest you split this PR in 2!

Reply:
Why?